.github/workflows/ci.yaml

```diff
@@ -699,6 +699,7 @@ jobs:
         with:
           repository: "sullo/nikto"
           path: nikto
+          ref: nikto-2.5.0
```

Comment:
I think it would be great to create a variable containing the desired version to minimize the locations we have to update for a new release